docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -38,6 +38,8 @@ \section{Problem Statement}
 
 \subsection{Problem}
 
+With the rising cost of living in Hamilton as well as across Canada, many households are searching for ways to cut back on costs of thier living expenses. Grocery prices, most notably, are quickly on the rise and are making it difficult for households to accommodate in their budgets. \textbf{Grocery Spend Tracker} will strive to make it easier for users to make \textit{smart} grocery decisions to lower their spending on food and household needs.
```

Comment:
Grocery prices, most notably, are quickly on the rise and are making it difficult for households to _handle/manage_ their budgets